apps/marketplace/components/BuyerSpecialist/BuyerSpecialistTimeframesAndBudgetStage.js

```diff
@@ -11,8 +11,10 @@ import DateControl from 'marketplace/components/BuyerBriefFlow/DateControl'
 
 const contractLengthRequired = v => required(v.contractLength)
 const startDateRequired = v => required(v.startDate)
-const startDateIsValid = v => validDate(v.startDate)
-const startDateIs2DaysInFuture = v => !startDateRequired(v) || !startDateIsValid(v) || dateIs2DaysInFuture(v.startDate)
+const startDateIsValid = v => !startDateRequired(v) || validDate(v.startDate, false)
+const startDateIsInPast = v => !startDateRequired(v) || !startDateIsValid(v) || validDate(v.startDate, true)
```

Comment:
@tobiasbrown I made the following changes as you requested. So now the code follows the pattern of the other functions.